### DISCUSSION TITLE
How to change the fonts

### DISCUSSION THREAD

**Discussion by TS-RM** [excalidraw/excalidraw-vscode#62] [Q&A]
How can we change the fonts for some language the fonts are not that much good, for example in Arabic its not good.

**Comment by Qiuxiaoye007** [comment]
我不知道如何修改代码部分，但是我发现在插件的文件目录中替换字体文件可以达到修改字体的效果，具体操作如下：
1. 找到 vscode 的中 Excalidraw 的插件目录中的资源存放文件夹，一般的路径在 `.....vscode\extensions\pomdtr.excalidraw-editor-3.7.4\public\excalidraw-assets`
2. 其中存在 6 中内置的默认字体文件，分别是
  - `Assistant` 字体
    - Assistant-Bold: 粗体版本，用于强调内容，例如标题或需要突出的文本部分。
    - Assistant-Medium: 中等粗细，适合副标题或稍微需要强调的内容。
    - Assistant-Regular: 常规版本，通常用于正文文本，提供良好的可读性。
    - Assistant-SemiBold: 半粗体版本，适合介于普通正文和强调内容之间的应用场景。
  - `Cascadia.woff2`：代码字体
  - `Virgil.woff2`：手写字体
3. 将你的字体重命名为上述 6 款中需要修改的那一款即可
